docs/design/keyboard-shortcuts.md

```diff
@@ -210,7 +210,7 @@ The user can select which action the keyboard shortcut will take. After making t
 For the best user experience, we recommend that you minimize conflicts with Excel with these good practices.
 
 - Use only keyboard shortcuts with the following pattern: **Ctrl+Shift+Alt+*x***, where *x* is some other key.
-- If you need more keyboard shortcuts, check the [list of Excel keyboard shortcuts](https://support.microsoft.com/office/keyboard-shortcuts-in-excel-1798d9d5-842a-42b8-9c99-9b7213f0040f), and avoid using any of them in your add-in.
+- If you need more keyboard shortcuts, check the [list of Excel keyboard shortcuts](https://support.microsoft.com/office/1798d9d5-842a-42b8-9c99-9b7213f0040f), and avoid using any of them in your add-in.
```

Comment:
@Rick-Kirkham For awareness about this link style change, since he's working on this article in a separate PR.